networking/ipv6-control-plane.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Configure Kubernetes control plane to operate over IPv6
+description: Configure the Kubernetes control plane to operate over IPv6 for dual stack or IPv6 only.
+canonical_url: '/networking/ipv6-control-plane'
+---
+
+### Big picture
+
+Depending on your networking set up, you may want to configure the Kubernetes control plane to operate over IPv6 for dual stack or IPv6 only.
+
+### How to
+ 
+To configure Kubernetes components for IPv6 only, set the following flags. 
+
+| Component   | **Flag**        | **Value/Content**                                            |
+| ----------- | --------------- | ------------------------------------------------------------ |
+| **kubelet** | `--address`     | Set to the appropriate IPv6 address or `::` for all IPv6 addresses. |
+|             | `--cluster-dns` | Set to the IPv6 address that will be used for the service DNS; this must be in the range used for `--service-cluster-ip-range`. |
+|             | `--node-ip`     | Set to the IPv6 address of the node.                         |
```

Comment:
I'm sorry, I think my previous comments were unclear, but we have lost too much material here.  I believe we should have all this:
```
| Component                   | **Flag**                                      | **Value/Content**                                            |
| --------------------------- | --------------------------------------------- | ------------------------------------------------------------ |
| **kube-apiserver**          | `--bind-address` or `--insecure-bind-address` | Set to the appropriate IPv6 address or `::` for all IPv6 addresses on the host. |
|                             | `--advertise-address`                         | Set to the IPv6 address that nodes should use to access the `kube-apiserver`. |
| **kube-controller-manager** | `--master`                                    | Set with the IPv6 address where the `kube-apiserver` can be accessed. |
| **kube-scheduler**          | `--master`                                    | Set with the IPv6 address where the `kube-apiserver` can be accessed. |
| **kubelet**                 | `--address`                                   | Set to the appropriate IPv6 address or `::` for all IPv6 addresses. |
|                             | `--cluster-dns`                               | Set to the IPv6 address that will be used for the service DNS; this must be in the range used for `--service-cluster-ip-range`. |
|                             | `--node-ip`                                   | Set to the IPv6 address of the node.                         |
| **kube-proxy**              | `--bind-address`                              | Set to the appropriate IPv6 address or `::` for all IPv6 addresses on the host. |
|                             | `--master`                                    | Set with the IPv6 address where the `kube-apiserver` can be accessed. |
```

Reply:
fixed.